src/unsafe-deep-dive/foundations/what-is-unsafe.md

```diff
@@ -0,0 +1,89 @@
+# What is &ldquo;unsafety&rdquo;?
+
+Rust is intentionally vague about what "unsafe" means.
+
+Let's create a working definition together.
+
+<details>
+
+## Clarifying "intentionally vague"
+
+It's likely that you'll have an audience member object to the phrase
+"intentionally vague". Perhaps refer to this statement from the standard library
+when discussing the [safety requirements of raw pointers].
+
+> Many functions in [this module] take raw pointers as arguments and read from
+> or write to them. For this to be safe, these pointers must be _valid_ for the
+> given access.
+>
+> ...
+>
+> The precise rules for validity are not determined yet.
+
+[this module]: https://doc.rust-lang.org/std/ptr/index.html
+[safety requirements of raw pointers]: https://doc.rust-lang.org/std/ptr/index.html#safety
+
+## Definitions from authoritative docs:
+
+stdlib's keyword:
```

Comment:
Sorry, I don't understand what it means because the keyword is a part of the language, not of the stdlib (this looks like a categorical error to me).